README.md

```diff
@@ -6,9 +6,9 @@
 
 Site content is written in Markdown format located in:
 
-- [`docs/`](/docs/) for the documentation;
-- [`blog/`](/blog/) for the blog;
-- [`src/`](/src/) for the pages;
+- [`docs/`](/docs/) for the [documentation](https://yourls.org/docs) ([development guide](https://docusaurus.io/docs/docs-introduction));
+- [`blog/`](/blog/) for the [blog](https://yourls.org/blog) ([development guide](https://docusaurus.io/docs/blog));
+- [`src/`](/src/) for the [pages](https://yourls.org/) ([development guide](https://docusaurus.io/docs/creating-pages)).
```

Comment:
Steal one word from Docusaurus docs and this will be a bit clearer

```suggestion
- [`src/`](/src/) for the [standalone pages](https://yourls.org/) ([development guide](https://docusaurus.io/docs/creating-pages)).
```